client/components/tinymce/plugins/mentions/suggestion-list.jsx

```diff
@@ -10,82 +10,72 @@ import PopoverMenu from 'components/popover/menu';
 import PopoverMenuItem from 'components/popover/menu-item';
 import Suggestion from './suggestion';
 
-const SuggestionList = React.createClass( {
-	displayName: 'SuggestionList',
+const getMatchingSuggestions = function( suggestions, query ) {
+	const matcher = new RegExp( '^' + query + '| ' + query, 'ig' ); // Start of string or preceded by a space.
+	let matchingSuggestions = suggestions;
 
-	propTypes: {
-		query: React.PropTypes.string,
-		suggestions: React.PropTypes.array,
-		isVisible: React.PropTypes.bool,
-		onClick: React.PropTypes.func,
-		onClose: React.PropTypes.func
-	},
+	if ( query.length > 0 ) {
+		matchingSuggestions = [];
 
-	getDefaultProps() {
-		return {
-			query: '',
-			suggestions: [],
-			isVisible: false
-		};
-	},
+		for ( let i = 0, len = suggestions.length; i < len; i++ ) {
+			const suggestion = suggestions[ i ];
+			const name = suggestion.name || suggestion.user_login + ' ' + suggestion.display_name;
 
-	getMatchingSuggestions() {
-		const { suggestions, query } = this.props;
-		const matcher = new RegExp( '^' + query + '| ' + query, 'ig' ); // Start of string or preceded by a space.
-		let matchingSuggestions = suggestions;
-
-		if ( query.length > 0 ) {
-			matchingSuggestions = [];
-
-			for ( let i = 0, len = suggestions.length; i < len; i++ ) {
-				const suggestion = suggestions[ i ];
-				const name = suggestion.name || suggestion.user_login + ' ' + suggestion.display_name;
-
-				if ( name.toLowerCase().match( matcher ) ) {
-					matchingSuggestions.push( suggestion );
-				}
+			if ( name.toLowerCase().match( matcher ) ) {
+				matchingSuggestions.push( suggestion );
 			}
 		}
+	}
+
+	return matchingSuggestions.slice( 0, 10 );
+};
 
-		return matchingSuggestions.slice( 0, 10 );
-	},
+const SuggestionList = ( { isVisible, suggestions, query, popoverContext, onClose, onClick } ) => {
+	if ( ! isVisible || ! suggestions ) {
+		return null;
+	}
 
-	render() {
-		const { isVisible, suggestions, query, popoverContext, onClose, onClick } = this.props;
+	const matchingSuggestions = getMatchingSuggestions( suggestions, query );
 
-		if ( ! isVisible || ! suggestions ) {
-			return null;
-		}
+	if ( matchingSuggestions.length > 0 ) {
+		return (
+			<PopoverMenu
+				className="mentions__suggestions"
+				context={ popoverContext }
+				isVisible={ isVisible }
+				onClose={ onClose }>
+					{ matchingSuggestions.map( ( suggestion ) => {
+						return (
```

Comment:
fun fact! this `return` isn't necessary. if all we're doing in a fat-arrow function is returning a value, we don't need to create the curlies

```js
{ matchingSuggestions.map( suggestion => <div>{ suggestion }</div> ) }

// or with multiline values I like wrapping in parentheses instead

{ matchingSuggestions.map( suggestion => (
	<div>
		<p>{ suggestion }</p>
	</div>
) }
```